install_config/monitoring/configuring-etcd-monitoring.adoc

```diff
@@ -185,3 +185,13 @@ $ oc apply -f etcd-cert-secret.yaml
 . Now that you have configured authentication, visit the *Targets* page of the web interface again. Verify that `etcd` is now being correctly monitored. It might take several minutes for changes to take effect.
 +
 image::etcd-monitoring-working.png[]
+
+. If you want `etcd` monitoring to be automatically updated when you update {product-title}, set this variable in the Ansible inventory file to `true`:
++
+`openshift_cluster_monitoring_operator_etcd_enabled`
```

Comment:
Not critical, but this variable looks a little odd on its own line like this. I'd suggest to put it in a code block, like:

```
----
openshift_cluster_monitoring_operator_etcd_enabled=true
----
```